distribution/build.gradle

```diff
@@ -227,13 +227,16 @@ configure(subprojects.findAll { ['archives', 'packages'].contains(it.name) }) {
      *                   Common files in all distributions                       *
      *****************************************************************************/
     libFiles = copySpec {
-      into 'lib'
-      from { project(':server').jar }
-      from { project(':server').configurations.runtime }
-      from { project(':libs:plugin-classloader').jar }
-      // delay add tools using closures, since they have not yet been configured, so no jar task exists yet
```

Comment:
This comment is important to keep in some form. It is the reason all these `from` statements use a closure.

Reply:
I added this comment back.